makers/staff_engineer.md

```diff
@@ -1,47 +1,78 @@
-* A reputation for submitting maintainable code that is well tested for review and QA
-* A reputation for launching quality, turbulence-free code and projects
-* The ability to balance multiple goals across tech/product/strategy/other teams and concerns
-* The ability to communicate effectively with non-engineers of varying levels at the company
-* Stepped up to take leadership of a project or team when needed
-
-### Is expected to
-
-* Be accountable for design and architecture direction by providing feedback and mentoring other engineers within their brand and working with principal engineers and architects
-* Plan, sequence, forecast, and track projects effectively across the brand
-* Work with senior managers & directors to ensure engineering effort is focussed on the most impactful areas for their brand
-* Monitor technical performance of engineers in a brand, and partner with engineering managers and leads on mentoring and growth
-* Develop and constantly refine technical, leadership, and team fit assessments (coding exercises, interview questions, etc) for interviews across the Engineering organization
-* Lead code/design/architecture reviews across teams and mentor & coach engineers to ensure work follows best practices & standards. Create and evolve brand wide practices and share with the rest of the organization.
-* Ensure that all brand decisions are in service of their customers
-* Help brand teammates communicate their design and architecture decisions throughout the company
-* Acts as a mentor to other senior and staff engineers who are developing their own mentees
-* Provide task definition and design/architecture direction for other engineers
-* Demonstrate expert knowledge of their discipline
-* Remain flexible & put customer needs first 
-* Accurately estimate time to complete projects given structure of team and deliver on those estimates
-* Allocate people within a team to maximise output of useful and team-goal-focused engineering work
-* Monitor performance of engineers on team, and partner with engineering managers on mentoring and growth
-* Look for ways to reduce distractions, interruptions, and unplanned work for themselves and others 
-* Challenge their team to demonstrate greater sense of urgency
-* Model bringing solutions to problems (not surfacing problems alone) and teach others on their team how to do the same
-* Be active in the technology community, positively contribute to Mariana Tek’s reputation, and find great candidates to grow the Mariana Tek team
-* Develop and constantly refine technical and team fit assessments (coding exercises, interview questions, etc) for interviews
-* Ensure code/design/architecture reviews happen on teams and that work follows established best practices and standards
-* Coordinate evaluation of tools and frameworks for the engineering team
-* Take responsibility for actions of their team, good and bad
+<!-- TODO: Add summary -->
+
+## Qualifications
```

Comment:
I feel this captures the role well - looking to @cathrynsupko here as well for a gut check. I think the balance between what we are tasked with both technically and non-technically feels pretty accurate to me, TBH.